apps/backoffice/src/lib/be/azure-access-token.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  AccessToken,
+  AzureAuthorityHosts,
+  ClientSecretCredential
+} from "@azure/identity";
+import { readableReport } from "@pagopa/ts-commons/lib/reporters";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import * as E from "fp-ts/lib/Either";
+import * as t from "io-ts";
+import { cache } from "react";
+import { HealthChecksError } from "./errors";
+
+//Instanza contentente le credenziali per accedere ad Azure
+let accessToken: AccessToken;
+
+type Config = t.TypeOf<typeof Config>;
+const Config = t.type({
+  AZURE_CLIENT_SECRET_CREDENTIAL_CLIENT_ID: NonEmptyString,
+  AZURE_CLIENT_SECRET_CREDENTIAL_SECRET: NonEmptyString,
+  AZURE_CLIENT_SECRET_CREDENTIAL_TENANT_ID: NonEmptyString,
+  AZURE_CREDENTIALS_SCOPE_URL: NonEmptyString
+});
+
+const getAzureConfig: () => Config = cache(() => {
+  const result = Config.decode(process.env);
+
+  if (E.isLeft(result)) {
+    throw new Error(
+      `error parsing azure access-token config, ${readableReport(result.left)}`
+    );
+  }
+  return result.right;
+});
+
+const refreshAzureAccessToken = async (): Promise<AccessToken> => {
+  const apimConfig = getAzureConfig();
+
+  let tokenResponse: AccessToken;
+  try {
+    const credential = new ClientSecretCredential(
+      apimConfig.AZURE_CLIENT_SECRET_CREDENTIAL_TENANT_ID,
+      apimConfig.AZURE_CLIENT_SECRET_CREDENTIAL_CLIENT_ID,
+      apimConfig.AZURE_CLIENT_SECRET_CREDENTIAL_SECRET,
+      {
+        authorityHost: AzureAuthorityHosts.AzurePublicCloud
+      }
+    );
+    tokenResponse = await credential.getToken(
+      apimConfig.AZURE_CREDENTIALS_SCOPE_URL
+    );
+  } catch (e) {
+    console.log("Error retrieving on Azure Access Token", e);
```

Comment:
**suggestion:** log this event as error level

Reply:
done.
BTW this module has been removed

Reply:
done